docs/cosign.md

```diff
@@ -0,0 +1,31 @@
+# Container Image Sign and Verify with cosign tool
+
+[cosign](https://github.com/sigstore/cosign) is tool that allows you to sign and verify container images with the
+public/private key pairs or without them by providing
+a [Keyless support](https://github.com/sigstore/cosign/blob/main/KEYLESS.md).
```

Comment:
"Keyless" needs to be briefly explained.

Reply:
Done.